sw/device/tests/alert_handler_lpg_clkoff_test.c

```diff
@@ -375,13 +377,16 @@ void set_peripheral_clock(const test_t *peripheral,
           "intended_clk_state = %d, received_clk_state = %d", new_clk_state,
           clk_state);
   }
+  if (new_clk_state == kDifToggleDisabled) {
+    LOG_INFO("Turn off %0s clock", peripheral->name);
+  }
 };
```

Comment:
When I use `%0s`, it prints "<BAD argument>" on FPGA. Using `%s` fixes the problem and prints the correct peripheral name.